.github/scripts/test_tekton_tasks.sh

```diff
@@ -88,14 +88,20 @@ do
     continue
   fi
 
+  # Use a copy of the task file to prevent modifying to original task file
+  TASK_COPY=$(mktemp)
+  cp "$TASK_PATH" "$TASK_COPY"
+
   if [ -f ${TESTS_DIR}/pre-apply-task-hook.sh ]
   then
     echo Found pre-apply-task-hook.sh file in dir: $TESTS_DIR. Executing...
-    ${TESTS_DIR}/pre-apply-task-hook.sh
+    ${TESTS_DIR}/pre-apply-task-hook.sh "$TASK_COPY"
   fi
 
   echo "  Installing task"
-  kubectl apply -f $TASK_PATH
+  kubectl apply -f "$TASK_COPY"
+
+  rm "$TASK_COPY"
```

Comment:
do we need `-f`?

Reply:
Good point, added. Traditionally, root users would have an alias like `alias rm="rm -i"`. I just checked and it's not the case on my system (macOS). And it's likely not the case in the github ci environment because all the checks passed - including running many task tests. But it's still probably better to have it.